src/components/UserLoginPage.vue

```diff
@@ -63,7 +63,7 @@ export default {
         password: this.password,
       }).then(() => {
         this.$router.push({
-          path: this.$route.query.redirect || '/',
+          path: window.previousUrl || window.redirect || '/',
```

Comment:
Hi @PaulSchroeder why did you change it with window properties? It should be better to use the route plugin indeed...